sage-ios/SAGE/SAGE/StartSemesterController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  StartSemesterController.swift
+//  SAGE
+//
+//  Created by Andrew Millman on 1/17/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class StartSemesterViewController: UIViewController {
+    
```

Comment:
inherit from the formcontroller class i made in my last pr, it'll do the finish button and call the completeForm method for you